ServiceInnovationLab/pancake-frontend
# prevent user from typing commars in wage or salary income field

# What has changed and why?

closes #502 

Added check if income is a number
Refactored duplicate methods.

# How to test this change
Fill out form on first page.
On the second page, scroll to the income section, select 'Wage or Salary' and test that commars can not be typed in the input field.

Also make sure we cannot enter negative values

- [ ] tested in small screen
- [ ] tested in IE
- [ ] tested in screen reader/contrast <-- remove this if no UX change
